uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/selectpicker/ElementHelper.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.uberfire.client.views.pfly.selectpicker;
+
+import elemental2.dom.Node;
+
+public class ElementHelper {
```

Comment:
Just a question - why were these classes moved from kie-wb-common to appformer?

Reply:
Because it is a generic class that can be used in other places, and not just in dmn. I was moving the other classes in this package here for the same reason, and to be able to use them on Library. I've just took advantage of the opportunity :).